swaywm/wlroots
# Decorations

Add decorations to rootston xwayland views.

Feedback on the approach would be great.

Tests:

* mouse over decoration borders and the correct cursor should show up
* start an interactive move by dragging the titlebar
* start an interactive resize by dragging the borders
* try again with the view rotated
